tt_metal/distributed/mesh_device.cpp

```diff
@@ -115,11 +115,15 @@ MeshShape SystemMesh::Impl::get_system_mesh_shape(size_t system_num_devices) {
 
 std::unordered_map<LogicalCoordinate, PhysicalCoordinate> SystemMesh::Impl::get_system_mesh_translation_map(
     size_t system_num_devices) {
+    std::string galaxy_mesh_descriptor = "TG.json";
+    if (tt::Cluster::instance().number_of_pci_devices() == system_num_devices) {
```

Comment:
nit: leave a comment explaining this? I'm looking at this for the first time, and a bit lost here.

Reply:
It's to differentiate Quanta vs. old-style galaxy systems by looking at number of PCIE devices. Agree about adding a comment.